src/docs/Components/Form/Radio/index.mdx

```diff
@@ -11,6 +11,67 @@ status:
 
```

Comment:
The "definition" is very weird to me. It doesn't define what it is or what it does but it's being opposed to checkboxes. Can we try something like: "A radio button is used to offer a unique choice to your user in a form."